.pre-commit-config.yaml

```
@@ -58,7 +58,7 @@ repos:
           - -d
           - "{extends: relaxed, rules: {line-length: {max: 90}}}"
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.4.10
+    rev: v0.5.0
```

```suggestion
     rev: v0.4.10
```
That's a brakign change. Defer to a PR of it own.